README.md

```diff
@@ -1,2 +1,25 @@
 # QuickFix
-bref décrire l'appli ici
+
+**QuickFix** is a platform designed to connect users who need tasks done with skilled manual workers. Whether it's painting, plumbing, or any other service, QuickFix makes it easy to find the right person for the job.
+
+## Features
+- **User-friendly Interface**: Easy-to-navigate platform for users and workers alike.
+- **Worker Profiles**: Manual workers can create detailed profiles showcasing their experience, services, and rates.
+- **Rating & Reviews**: Users can leave feedback for workers after a job is completed, ensuring quality and trust in the system.
```

Comment:
It would be better to highlight the fact that the app have two different interfaces for each the clients and the workers

Reply:
Done :)